numpy/core/tests/test_print.py

```diff
@@ -35,7 +35,7 @@ def test_float_types():
     """ Check formatting.
 
         This is only for the str function, and only for simple types.
-        The precision of np.float and np.longdouble aren't the same as the
```

Comment:
Only manual lines in this patch, to avoid the claim that `float` is not the same as `float`. Of course, the original was wrong too.